src/main/java/com/salesforce/storm/spout/sideline/handler/SidelineSpoutHandler.java

```diff
@@ -429,17 +429,12 @@ public void stopSidelining(SidelineRequest sidelineRequest) {
             );
 
```

Comment:
This is just duplicated code.  We were checking to see if sidelinePayload was equal to null twice.  Didn't make sense, because it can't be null after we check it the first time.  I just replaced the top check with the bottom check.  Only real difference is its now an error instead of warn, and logs more details.